src/components/BookList.js

```diff
@@ -11,28 +13,48 @@ const BookList = () => {
   };
 
   return (
-    <div>
-      <h3>List of Books</h3>
+    <div className="bookList">
       <ul>
         {Object.entries(books).map(([id, book]) => book.map((bookItem) => (
-          <li key={id}>
-            <div>
+          <li key={id} className="book-item">
+            <div className="book-details">
               <p>{bookItem.category}</p>
               <h2>{bookItem.title}</h2>
-              <p>{bookItem.author}</p>
-              <div>
+              <p className="author">{bookItem.author}</p>
+              <div className="book-buttons">
                 <button
                   type="button"
                   onClick={() => {
                     handleDelete(id);
                   }}
+                  className="bookButtons"
                 >
                   delete
                 </button>
+                <button className="bookButtons" type="button">
+                  edit
+                </button>
+                <button className="comment" type="button">
+                  comment
+                </button>
```

Comment:
- [ ] Please style the book cards to look exactly like the zeplin design. To achieve this, you have to do the following: 👇 
1. Use the specified `font-family`
2. Use the specified colors as required
3. Add a vertical border to the right just before the CURRENT CHAPTER section
4. Style the button just as specified with the appropriate color, `font-family` and `font-weight`
Please see the annotated diagram below for more information 👇 
### Zeplin Design
![image](https://github.com/berylcl/bookstore/assets/46839548/6fdab960-e093-4551-bef3-c905915d7350)

### Your Implementation
![image](https://github.com/berylcl/bookstore/assets/46839548/29ac3ccd-6285-4211-b790-d9bf27da222d)